README.md

```diff
@@ -1,14 +1,10 @@
 # WooCommerce Services
 
-WooCommerce Services development is now in BETA - this means that you should expect frequent and sometimes large changes as we improve the service. During BETA testing, we'll be fixing problems and working to launch a feature-complete MVP.
-
-**This is a BETA release. Please carefully test and make sure it meets your needs before considering for a production site.**
-
 WooCommerce Services makes basic eCommerce features like shipping more reliable by taking the burden off of your site’s infrastructure.
 
 With WooCommerce Services, critical services are hosted on Automattic’s best-in-class infrastructure, rather than relying on your store’s hosting. That means your store will be more stable and faster.
 
-The emphasis for initial release of WooCommerce Services is shipping simplified. We are providing Shipping Zones-compatible USPS and Canada Post shipping methods and USPS shipping labels (stamps). Shipping Zones are [an exciting new feature of WooCommerce 2.6](https://woocommerce.wordpress.com/2016/02/10/shipping-zones-to-ship-with-2-6/).
+The emphasis for initial release of WooCommerce Services is shipping simplified. We are providing Shipping Zones-compatible USPS and Canada Post shipping methods and USPS shipping labels (stamps). Shipping Zones are [an exciting new feature of WooCommerce 2.6](https://woocommerce.wordpress.com/2016/02/10/shipping-zones-to-ship-with-2-6/). We are also allowing to print discounted USPS shipping labels.
```

Comment:
Oh my bad, I didn't realize the labels stuff was already in the previous version of this paragraph. I'll remove so much emphasis in Shipping Zones and leave the rest as it was.